builder/vsphere/supervisor/step_validate_import_test.go

```diff
@@ -0,0 +1,265 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package supervisor_test
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/packer-plugin-sdk/multistep"
+	imgregv1a1 "github.com/vmware-tanzu/image-registry-operator-api/api/v1alpha1"
```

Comment:
This should be consistent. WOuld not recommend the a1.

`imagev1` or `imageRegistryv1`

Reply:
changed to imgregv1